### PR TITLE
[DET-1235] Upgrade to Airflow 1.10.15.

### PR DESCRIPTION
**Pull request by fpopic**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-local_scripts.md
-.idea/*
\ No newline at end of file
+*.iml
```
Comment (fpopic) [inline .gitignore:1]:
```suggestion
local_scripts.md
*.iml
```

**Review by bpietraga — APPROVED** [review]
tenor-131715404

File: Dockerfile [hunk 2]
```diff
@@ -46,14 +45,17 @@ libpq-dev \
         netcat \
         rsync \
     && sed -i 's/^# en_US.UTF-8 UTF-8$/en_US.UTF-8 UTF-8/g' /etc/locale.gen \
-    && locale-gen \
-    && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
-    && pip install "pip==21.1.1" \
+    && locale-gen en_US.UTF-8 \
+    && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8
```
Comment (fpopic) [inline Dockerfile:49]: was getting warning about this when building

https://www.thomas-krenn.com/en/wiki/Perl_warning_Setting_locale_failed_in_Debian

**Review by bpietraga — APPROVED** [review]

**Comment by fpopic** [comment]
We are lucky, somebody in puckel's image just did PR to upgrade to 1.10.14 so we should compare and handle leftovers 

https://github.com/puckel/docker-airflow/pull/639/files

**Pull request closed without merging**